propolis/src/hw/ps2ctrl.rs

```diff
@@ -652,6 +668,10 @@ impl PS2Mouse {
     fn reset(&mut self) {
         // XXX  what should the defaults be?
         self.buf.clear();
+        self.cur_cmd = None;
```

Comment:
`PS2Kbd::reset` doesn't clear `cur_cmd` in its `reset` method, should that also be updated to match `PS2Mouse` here?

Reply:
Mm, yeah.  Good catch.